syntax = "proto3";

message product {
  string product_name = 1;
  int32 product_id = 2;
  string product_desc = 3;
}
